packages/themes/src/white.js

```diff
@@ -304,6 +304,12 @@ export {
   container03,
   container04,
   container05,
+  sizeXSmall,
```

Comment:
If we are adding this to the `white` theme, does it need to be added to each theme?